mite/context.py

```diff
@@ -53,25 +54,37 @@ def should_stop(self):
             return self._should_stop_func()
         return False
 
+    @property
+    def transaction_name(self):
+        return active_transaction.get()[0]
+
+    @property
+    def transaction_id(self):
+        return active_transaction.get()[1]
+
+    def extend_transaction(self, name):
```

Comment:
Same: I think this should be prefixed with an underscore and (thus) not be part of the public api of Context